apps/frontend/src/pages/CreateListing.tsx

```diff
@@ -76,10 +74,16 @@ const CreateListing = () => {
     submissionEvent,
   ) => {
     submissionEvent.preventDefault();
-    if (!priceError && !titleError && !sent) {
+    console.log(newListingObject);
+    if (!titleError && !sent) {
       // In order to make sure that the images are retrieved before submitting
       const successImages: boolean = await asyncListingImageWrapper();
 
+      if (!newListingObject.listing.title) {
+        setTitleError("Please enter a title");
```

Comment:
I may have missed it, but is there any error shown for not entering a price?

Reply:
So I got rid of it because I have it registering an empty field as 0, I also made it so the input cannot be anything but acceptable values, if think I should add it back though, that shouldn't be much of an issue!